build.xml

```diff
@@ -32,6 +32,19 @@
 		</exec>
 	</target>
 
+	<target name="composer-normalize-check">
+		<exec
+				executable="composer"
+				logoutput="true"
+				passthru="true"
+				checkreturn="true"
+		>
+			<arg value="normalize"/>
+			<arg value="--ansi"/>
```

Comment:
It's inconsistent with the other commands, but I added the `--ansi` flag to ensure colored output during the execution. This makes it easier to spot the difference between normalize and not normalized, for example:

![screen shot 2018-07-21 at 10 02 31](https://user-images.githubusercontent.com/605483/43033648-443524be-8ccd-11e8-85e7-60686f70c57c.png)

Reply:
Cool, I'll welcome other PR that adds `--ansi` to compatible commands 😊

Reply:
@ondrejmirtes All Symfony console commands support `--ansi` 🙂